<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>citycollectionwidget</class>
 <widget class="QWidget" name="citycollectionwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbLeftUpIcon">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>15</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbLeftUpTitle">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>18</y>
     <width>56</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>6</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbCityCurrent">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>66</y>
     <width>72</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbCityCollect">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>213</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbCityCount">
   <property name="geometry">
    <rect>
     <x>516</x>
     <y>221</y>
     <width>19</width>
     <height>11</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="backwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>560</height>
    </rect>
   </property>
  </widget>
  <zorder>backwidget</zorder>
  <zorder>lbLeftUpIcon</zorder>
  <zorder>lbLeftUpTitle</zorder>
  <zorder>btnCancel</zorder>
  <zorder>lbCityCurrent</zorder>
  <zorder>lbCityCollect</zorder>
  <zorder>lbCityCount</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
